display/pfobjects.py

```diff
@@ -116,8 +123,12 @@ def draw(self, projection, opt=''):
             
 class GStraightTrajectory(GTrajectory):
     def __init__(self, description):
-        super(GStraightTrajectory, self).__init__(description,
-                                                  linestyle=2, linecolor=1)
+        self.desc = description
+        if abs(self.desc.pdgid()) in [22]:
```

Comment:
use the if to define linestyle and color, and move the call to super after the if block